### DISCUSSION TITLE
Qdrant on our own GCP account

### DISCUSSION THREAD

**Discussion by navneetrajput06** [qdrant/qdrant#3211] [Q&A]
We are going to use Qdrand for our vector application. Could you please answer our below questions?.
-- We need to setup qdrant on our own GCP account. we will be running pods using GKE.
-- What does node mean in QDrant ? Does different services run on different nodes and what is the criteria of setting the nodes in QDrant cluster.
-- Since we will be using this for production also we need to understand the consent of nodes and memory utilization.
-- What is the meaning of having multiple replications? it's just for additional data replication or it improves the processing also.
-- Kindly share the documentation for Qdrant cluster mode deployment with our own GCP using GKE.

**Comment by wisebreadloaf** [comment]
I am having the same issue where the client prefers their own gcp environment support with the vector db due to security concerns,are there any solutions???
